internal/osutil/sys/sysnum_16_linux.go

```diff
@@ -1,4 +1,5 @@
 // -*- Mode: Go; indent-tabs-mode: t -*-
+//go:build arm || 386
```

Comment:
These are due to the new `//go:build` style build constraints being introduced (and `go fmt` adds them in Go 1.17). See https://go.googlesource.com/proposal/+/master/design/draft-gobuild.md